sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -0,0 +1,9 @@
+import HttpClient from './HttpClient';
+
+export default abstract class HttpBaseClient
+{
+  protected apiClient = HttpClient;
+  handleRequestError(e:any){
+    console.log(e);
```

Comment:
may be we should add a technical debt for logging framework instead of console.logging

Reply:
Yes, we can. I am gonna create one. The idea here was just to introduce an idea for a base class to support HTTP API calls since this was the first story executing an API call.